packages/app/src/cli/services/build/extension.ts

```diff
@@ -141,11 +142,18 @@ export async function buildFunctionExtension(
   }
 
   try {
+    const bundlePath = joinPath(extension.outputPath.split('/').slice(0, -2).join('/'), 'index.wasm')
+    extension.outputPath = joinPath(extension.directory, joinPath('dist', 'function.wasm'))
```

Comment:
Yes, `outputPath` ends up becoming `<bundlepath>/dist/something.js` and since `<bundlepath>/dist` does not exists it complains `No such file or directory` (remember that bundlePath is a /tmp directory). So, I essentially set it to the valid path that exists which is `<extension directory>/dist`.